App.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import Login from './Login';
+import Dashboard from './Dashboard';
+
+function App() {
+  return (
+    <Router>
```

Comment:
To improve maintainability, consider abstracting the routing logic into a separate component. This will make the App component cleaner and more focused on its main responsibility.